docs/tutorials/chaos/slo-validation-chaos.md

```diff
@@ -0,0 +1,177 @@
+      name: httpbin
+      labels:
+        app: httpbin
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: httpbin
+      template:
+        metadata:
+          labels:
+            app: httpbin
+        spec:
+          containers:
+          - name: httpbin
+            image: kennethreitz/httpbin
+            ports:
+            - containerPort: 80
+          initContainers:
+          - name: init-myservice
+            image: busybox:1.28
+            command: ['sh', '-c', 'sleep 1']
+    EOF
+    ```
+    5. Create the `httpbin` deployment.
+    ```shell
+    kubectl apply -f deploy.yaml
+    ```
+    6. Create the `httpbin` service.
+    ```shell
+    kubectl expose deploy httpbin --port=80
+    ```
+
+***
+
+## Launch experiments
+Launch the LitmusChaos and Iter8 experiments as described below.
+=== "LitmusChaos"
+    ```shell
+    helm install httpbin litmuschaos \
+    --repo https://iter8-tools.github.io/hub/ \
+    --set applabel='app=httpbin' \
+    --set totalChaosDuration=3600 \
+    --set chaosInterval=5
+    ```
+
+    ??? note "About this LitmusChaos experiment"
+        This is a [LitmusChaos pod-delete experiment](https://litmuschaos.github.io/litmus/experiments/categories/pods/pod-delete/) packaged for reusability in the form of a Helm chart. This experiment causes (forced/graceful) pod failure of specific/random replicas of application resources, in this case, pods with a label called `app` with value `httpbin`.
+
+        The deletion of pod(s) will be attempted by the chaos experiment once every  `chaosInterval`(5) seconds, and this experiment will terminate after `totalChaosDuration`(3600) seconds.
+
+=== "Iter8" 
+    ```shell
+    iter8 k launch \
+    --set "tasks={ready,http,assess}" \
+    --set ready.deploy=httpbin \
+    --set ready.service=httpbin \
+    --set ready.chaosengine=litmuschaos-httpbin \
+    --set ready.timeout=60s \
+    --set http.url=http://httpbin.default/get \
+    --set http.duration=30s \
+    --set http.qps=20 \
+    --set assess.SLOs.upper.http/latency-mean=50 \
+    --set assess.SLOs.upper.http/latency-p99=100 \
+    --set assess.SLOs.upper.http/error-count=0 \
+    --set runner=job
+    ```
+
+    ??? note "About this Iter8 experiment"
+        This Iter8 experiment is similar to [your first Iter8 experiment](../../getting-started/your-first-experiment.md) with some notable changes. The `ready` task in this experiment also checks if the `chaosengine` resource exists before it starts, and in addition to the mean latency and error count SLOs, it verifies that the 99th percentile latency is under 100 msec.
+
+*** 
+
+## Observe experiments
+Observe the LitmusChaos and Iter8 experiments as follows. The chaos and Iter8 experiments 
+
+=== "LitmusChaos"
+    Verify that the phase of the chaos experiment is `Running`.
+    ```shell
+    kubectl get chaosresults/litmuschaos-httpbin-pod-delete -n default \
+    -ojsonpath='{.status.experimentStatus.phase}'
```

Comment:
Is there supposed to be a space?

```suggestion
    -o jsonpath='{.status.experimentStatus.phase}'
```

Reply:
I verified that the tutorial works as written.